concordia/templates/home.html

```diff
@@ -26,17 +26,17 @@
                     {% endif %}
                 </ol>
                 <div class="carousel-inner">
-                    <div class="carousel-item active" data-title="Walt Whitman at 200" data-hero-text="Transcribe and review poetry, letters, and writings of Walt Whitman. Discover how he wrote and lived for yourself." data-link-url="{% url 'transcriptions:redirect-to-next-transcribable-asset' 'walt-whitman' %}">
-                        <img class="d-block w-100" src="{% static 'img/homepage-carousel/whitman.jpg' %}" alt="Walt Whitman in his younger years, a black and white engraving. Whitman with his right hand on his hip and his left in his pocket wearing a black rimmed hat.">
+                    <div class="carousel-item active" data-title="Memorial Day Challenge" data-hero-text="Make it meaningful by reviewing soldiers' stories of service and sacrifice." data-link-url="{% url 'transcriptions:redirect-to-next-reviewable-asset' 'civil-war-soldiers-disabled-but-not-disheartened' %}">
+                        <img class="d-block w-100" src="{% static 'img/homepage-carousel/mem-day-challenge.jpg' %}" alt="Ornate left-handed penmanship specimen submitted by Alfred D. Whitehouse in which photograph portrait of Whitehouse is surrounded by caligraphy dedicating his work to his Union regiment. Whitehouse has a mustache and sideburns and wears a suit and bowtie.">
                     </div>
                     <div class="carousel-item" data-overlay-position="top-right" data-title="Review Now" data-hero-text="Approve or correct other volunteers' transcriptions to help them cross the finish line." data-link-url="{% url 'transcriptions:redirect-to-next-reviewable-asset' 'mary-church-terrell-advocate-for-african-americans-and-women' %}">
                         <img class="d-block w-100" src="{% static 'img/homepage-carousel/review.jpg' %}" alt="Close up of hand-written text of Mary Church Terrell's first speech to the NAACP">
                     </div>
-                    <div class="carousel-item"  data-title="Where to start?" data-hero-text="Find instructions and help to get started in our Welcome Guide" data-link-url="{% url 'welcome-guide' %}">
```

Comment:
If you're going to want this carousel slide restored at any point in the future, I suggest saving the copy to Confluence page somewhere - getting it out of Git history is a pain.

Reply:
Thats a good idea. @lalgee we can create a page in confluence under site content to keep carousel slides so we can rotate them in and out without having to redoing or cherry picking from past commits.

Reply:
Agreed.  I wondered about this and had already grabbed the Jump in code just in case.  Made space for these on the page I just created w/carousel update instructions. - 
https://staff.loc.gov/wikis/display/CS/Carousel+updates

Reply:
Getting stuff out of Confluence is also going to be a problem since it'll require manual review to keep it in sync with other updates to that page. I'm wondering whether the right answer is not to remove slides but feature-flag them.